### PR TITLE
4 create html for one menu

### LINKED ISSUE #4: Create_HTML_for_one_menu

### PR DESCRIPTION
**Pull request by Marcelin97**
close #4

### CODE REVIEW

File: pages/a_la_francaise.html [hunk 1]
```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="fr">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta
+            name="description"
+            content="Ohmyfood le menu à la française"
```
Comment (Tutt-J) [inline pages/a_la_francaise.html]: Please write a longer description

File: pages/a_la_francaise.html [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+        />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>À la française Ohmyfood</title>
```
Comment (Tutt-J) [inline pages/a_la_francaise.html]: Please pu a - or a | between menu name and ohmyfood

File: pages/a_la_francaise.html [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+    <link rel="stylesheet" href="../assets/css/menus.css" />
+</head>
+<body>
+<!--body-->
+    <header id="header_francaise">
```
Comment (Tutt-J) [inline pages/a_la_francaise.html]: remove this id

File: pages/a_la_francaise.html [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+        </figure>
+    <!--end of page header-->
+    </header>
+    <main>
+    <!--main content-->
+        <figure>
+            <img src="../assets/images/restaurants/A_la_francaise_3936w.jpg" 
+            alt="Assiette fantaisie en gros plan">
+        </figure>
+        <div>
```
Comment (Tutt-J) [inline pages/a_la_francaise.html]: use section

File: pages/a_la_francaise.html [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+            <H1>À la française<i class="far fa-heart"></i></H1>
```
Comment (Tutt-J) [inline pages/a_la_francaise.html]: H1 ==> h1

File: pages/a_la_francaise.html [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+            
+            <section id="francaise_entree">
```
Comment (Tutt-J) [inline pages/a_la_francaise.html]: just universal id like entries

File: pages/a_la_francaise.html [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+                <h2>ENTRÉES</h2>
+                    <article>
+                        <div>
+                            <H3>Tarte de poulpe acidulé</H3>
```
Comment (Tutt-J) [inline pages/a_la_francaise.html]: H3 ==> h3

File: pages/a_la_francaise.html [hunk 8]
```diff
@@ -0,0 +1,158 @@
...
+                    <article>
+                        <div>
+                            <H3>Soupe à l'oignon</H3>
+                            <p>Revisitée</p>
+                        </div>
+                        <span>20€</span>
+                        <!--add animation-->
+                    </article>
+            </section>
+            <section id="francaise_plats">
```
Comment (Tutt-J) [inline pages/a_la_francaise.html]: just universal id like dishes

File: pages/a_la_francaise.html [hunk 9]
```diff
@@ -0,0 +1,158 @@
...
+                    <article>
+                        <div>
+                            <H3>Pigeonneau d'Illes-et-Vilaine</H3>
+                            <p>Sur son lit de gnocchis aux légumes</p>
+                        </div>
+                        <span>44€</span>
+                        <!--add animation-->
+                    </article>
+            </section>
+            <section id="francaise_desserts">
```
Comment (Tutt-J) [inline pages/a_la_francaise.html]: just universal id like desserts

File: pages/a_la_francaise.html [hunk 10]
```diff
@@ -0,0 +1,158 @@
...
+                        <div>
+                            <H3>Millefeuille croustillant</H3>
+                            <p>Myrtilles et pâte d'amande</p>
+                        </div>
+                        <span>23€</span>
+                        <!--add animation-->
+                    </article>
+            </section>
+        </div>
+        <div>
```
Comment (Tutt-J) [inline pages/a_la_francaise.html]: remove this div

**Review by Tutt-J — CHANGES_REQUESTED** [review]

File: pages/a_la_francaise.html (hunk 10 above)
Comment (Marcelin97) [reply pages/a_la_francaise.html]: i removed

File: pages/a_la_francaise.html (hunk 9 above)
Comment (Marcelin97) [reply pages/a_la_francaise.html]: I made the change

File: pages/a_la_francaise.html (hunk 8 above)
Comment (Marcelin97) [reply pages/a_la_francaise.html]: I made the change

File: pages/a_la_francaise.html (hunk 7 above)
Comment (Marcelin97) [reply pages/a_la_francaise.html]: I made the change for all h3

File: pages/a_la_francaise.html (hunk 6 above)
Comment (Marcelin97) [reply pages/a_la_francaise.html]: I made the change

File: pages/a_la_francaise.html (hunk 5 above)
Comment (Marcelin97) [reply pages/a_la_francaise.html]: I made the change H1 to h1

File: pages/a_la_francaise.html (hunk 4 above)
Comment (Marcelin97) [reply pages/a_la_francaise.html]: I made the change div to section

File: pages/a_la_francaise.html (hunk 3 above)
Comment (Marcelin97) [reply pages/a_la_francaise.html]: i removed this id on header

File: pages/a_la_francaise.html (hunk 2 above)
Comment (Marcelin97) [reply pages/a_la_francaise.html]: i put this "|" between menu name and brand name

File: pages/a_la_francaise.html (hunk 1 above)
Comment (Marcelin97) [reply pages/a_la_francaise.html]: i wrote a new description

File: pages/a_la_francaise.html [hunk 11]
```diff
@@ -0,0 +1,160 @@
...
+<html lang="fr">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta
+            name="description"
+            content="Ohmyfood le menu gastronomique du restaurant chic À la française"
+        />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>À la française | Menu Ohmyfood</title>
```
Comment (Tutt-J) [inline pages/a_la_francaise.html]: just À la française | Ohmyfood

File: pages/a_la_francaise.html [hunk 12]
```diff
@@ -0,0 +1,160 @@
...
+        </figure>
+    <!--end of page header-->
+    </header>
+    <main>
+    <!--main content-->
+        <figure>
+            <img src="../assets/images/restaurants/A_la_francaise_3936w.jpg" 
+            alt="Assiette fantaisie en gros plan">
+        </figure>
+        <section id="francaise_menu">
```
Comment (Tutt-J) [inline pages/a_la_francaise.html]: remove this id !

File: pages/a_la_francaise.html (hunk 10 above)
Comment (Tutt-J) [reply pages/a_la_francaise.html]: Sure ? ^^

**Review by Tutt-J — CHANGES_REQUESTED** [review]

**Review by Tutt-J — APPROVED** [review]

**Pull request merged**